TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -2273,7 +2273,8 @@ as discussed in <doc:Declarations#Extension-Declaration>.
 ## Protocol Declaration
 
 A *protocol declaration* introduces a named protocol type into your program.
-Protocol declarations are declared at global scope
+Protocol declarations are declared either at global scope
+or nested inside a nongeneric type or nongeneric function,
```

Comment:
I believe we can delete the second instance of "nongeneric" here.
```suggestion
or nested inside a nongeneric type or function,
```

Reply:
I'd prefer to keep this, to avoid any risk of ambiguity, since we're in the language reference.